scripts/errors.js

```diff
@@ -48,7 +41,14 @@ const USED_ERRORS = [
    */
   'AuthorizationErrors.AuthorizationError', // 31201
   'AuthorizationErrors.RateExceededError', // 31206
-  'AuthorizationErrors.PayloadSizeExceededError', // 31209
+  /**
+   * Note(mhuynh): This error code has collision with a different error,
+   * Reconnect attempt is not authorized. Do not generate this error for now,
+   * fix with VBLOCKS-3144.
+   */
+  // 'AuthorizationErrors.PayloadSizeExceededError', // 31209
```

Comment:
voice errors 1.7 is already out and supports this. Do we have a ticket that will address this comment?

Reply:
Yes, the ticket VBLOCKS-3144 is already on line 47 in the comment/note.

Reply:
🤦